flang/include/flang/Runtime/stop.h

```diff
@@ -29,6 +29,7 @@ NORETURN void RTNAME(ProgramEndStatement)(NO_ARGUMENTS);
 // Extensions
 NORETURN void RTNAME(Exit)(int status DEFAULT_VALUE(EXIT_SUCCESS));
 NORETURN void RTNAME(Abort)(NO_ARGUMENTS);
+void RTNAME(Backtrace)(NO_ARGUMENTS);
```

Comment:
```suggestion
NORETURN  void RTNAME(Backtrace)(NO_ARGUMENTS);
```

Reply:
Thank you for your suggestion. But I don't quite understand it, because as far as I know, this macro is used when the function will cause the program to terminate or will not return to the caller. The Backtrace built-in function itself does not cause the program to terminate, and this built-in function will return to the caller function to continue execution. Therefore, I don't think it is necessary to add this macro. Of course, if there is another reason, please let me know, and I will accept your suggestion.

Reply:
Ok if the backtrace returns then you don't need it.